### PR TITLE
docs: add release notes for 0.18.1

### PR DESCRIPTION
**Pull request by eecsliu**
Add release notes for 0.18.1.

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,67 @@
...
+   -  ``det slot disable``
+   -  ``/config``
+   -  ``/api/v1/master/config``
+   -  ``/api/v1/agents/:agent_id/enable``
+   -  ``/api/v1/agents/:agent_id/disable``
+   -  ``/agents/:agent_id/slots/:slot_id``
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/enable``
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/disable``
+
+-  Logging: The default Fluentbit version in all deployment modes is now 1.9.3, changed from 1.6.
```
Comment (dzhu) [inline docs/release-notes.txt:64]:
```suggestion
-  Logging: The default Fluent Bit version in all deployment modes is now 1.9.3, changed from 1.6.
```

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,67 @@
...
 
+Version 0.18.1
+==============
+
+**Release Date:** May 24, 2022
+
+**New Features**
+
+-  Web: Themes have been introduced and styles have been adjusted to support various themes. Theme
+   switching is currently limited to dark/light mode and is set first through OS level preferences,
```
Comment (dzhu) [inline docs/release-notes.txt:19]:
```suggestion
   switching is currently limited to dark/light mode and is set first through OS-level preferences,
```

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,67 @@
...
+   then through browser level preference. In-app controllers will be coming soon.
```
Comment (dzhu) [inline docs/release-notes.txt:20]:
```suggestion
   then through browser-level preference. In-app controllers will be coming soon.
```

File: docs/release-notes.txt [hunk 4]
```diff
@@ -8,6 +8,67 @@
...
+
+-  Add experimental support for recovering live trials on master process restart. Users can restart
+   the master (with updated configuration options or an upgraded software version), and the current
+   running trials will continue running using the original configuration and harness versions. This
+   requires the agent to reconnect within a configurable ``agent_reconnect_wait`` period. This is
+   only available for ``agent`` resource manager, and can be enabled for resource pools using the
```
Comment (dzhu) [inline docs/release-notes.txt:26]:
```suggestion
   only available for the ``agent`` resource manager, and can be enabled for resource pools using the
```

File: docs/release-notes.txt [hunk 5]
```diff
@@ -8,6 +8,67 @@
...
+   ``agent_reattach_enabled`` flag. May only be available for "patch" level releases.
+
+-  Web: A trial restart counter has been added to the experiment detail header for single trial
+   experiments. For multi-trial experiments, they are show in a new `Restarts` column under `Trials`
```
Comment (dzhu) [inline docs/release-notes.txt:30]:
```suggestion
   experiments. For multi-trial experiments, trial restart counts are shown in a new `Restarts` column in the `Trials`
```

File: docs/release-notes.txt [hunk 6]
```diff
@@ -8,6 +8,67 @@
...
+   then through browser level preference. In-app controllers will be coming soon.
+
+-  Add experimental support for recovering live trials on master process restart. Users can restart
+   the master (with updated configuration options or an upgraded software version), and the current
+   running trials will continue running using the original configuration and harness versions. This
+   requires the agent to reconnect within a configurable ``agent_reconnect_wait`` period. This is
+   only available for ``agent`` resource manager, and can be enabled for resource pools using the
+   ``agent_reattach_enabled`` flag. May only be available for "patch" level releases.
+
+-  Web: A trial restart counter has been added to the experiment detail header for single trial
```
Comment (dzhu) [inline docs/release-notes.txt:29]:
```suggestion
-  Web: A trial restart counter has been added to the experiment detail header for single-trial
```

File: docs/release-notes.txt [hunk 7]
```diff
@@ -8,6 +8,67 @@
...
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/enable``
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/disable``
+
+-  Logging: The default Fluentbit version in all deployment modes is now 1.9.3, changed from 1.6.
+
+**Bug Fixes**
+
+-  Web: Fix the user filtering for migrating from `0.17.15` to `0.18.0`.
+-  API: Fix an issue where the ``POST /users`` endpoint always returned an error instead of the
+   user's information. This error was returned even when the user was created successfully.
```
Comment (dzhu) [inline docs/release-notes.txt:70]:
```suggestion
   user's information, even when the user was created successfully.
```

File: docs/release-notes.txt [hunk 8]
```diff
@@ -8,6 +8,67 @@
...
+-  Web: Themes have been introduced and styles have been adjusted to support various themes. Theme
+   switching is currently limited to dark/light mode and is set first through OS level preferences,
+   then through browser level preference. In-app controllers will be coming soon.
+
+-  Add experimental support for recovering live trials on master process restart. Users can restart
+   the master (with updated configuration options or an upgraded software version), and the current
+   running trials will continue running using the original configuration and harness versions. This
+   requires the agent to reconnect within a configurable ``agent_reconnect_wait`` period. This is
+   only available for ``agent`` resource manager, and can be enabled for resource pools using the
+   ``agent_reattach_enabled`` flag. May only be available for "patch" level releases.
```
Comment (dzhu) [inline docs/release-notes.txt:27]:
```suggestion
   ``agent_reattach_enabled`` flag. May only be available for patch-level releases.
```

File: docs/release-notes.txt [hunk 9]
```diff
@@ -8,6 +8,67 @@
...
+
+   -  Reading master config.
+   -  Enabling or disabling an agent.
+   -  Enabling or disabling a slot.
+
+-  Logging: Ensure logs for very short tasks are not truncated in Kubernetes.
+
+**Breaking Changes**
+
+-  Security: The following routes and CLI commands will need admin privileges.
```
Comment (dzhu) [inline docs/release-notes.txt:49]:
```suggestion
-  Security: The following routes and CLI commands now need admin privileges.
```

File: docs/release-notes.txt [hunk 10]
```diff
@@ -8,6 +8,67 @@
...
+
+   -  ``det master config```
```
Comment (dzhu) [inline docs/release-notes.txt:51]: Maybe put the routes first, matching the description.

File: docs/release-notes.txt [hunk 11]
```diff
@@ -8,6 +8,67 @@
...
+   -  ``/api/v1/agents/:agent_id/disable``
+   -  ``/agents/:agent_id/slots/:slot_id``
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/enable``
+   -  ``/api/v1/agents/:agent_id/slots/:slot_id/disable``
+
+-  Logging: The default Fluentbit version in all deployment modes is now 1.9.3, changed from 1.6.
+
+**Bug Fixes**
+
+-  Web: Fix the user filtering for migrating from `0.17.15` to `0.18.0`.
```
Comment (dzhu) [inline docs/release-notes.txt:68]:
```suggestion
-  Web: Fix the user filtering for migrating from Determined 0.17.15 to Determined 0.18.0.
```

**Review by vishnu2kmohan — APPROVED** [review]
LGTM, thanks @eecsliu (and @dzhu)!

**Pull request merged**